elastic/kibana
# [Observability RAC] Improve alerts table columns

## Summary

Fixes https://github.com/elastic/kibana/issues/105227

Here's how it looks with the fixes:
- The content of the "duration" column is right-aligned.
- Units in the "duration" column are
- The "actions" column (on the very left) has the headline "Actions".
- Labels are internationalized
<img width="1178" alt="Screenshot 2021-07-19 at 10 09 59" src="https://user-images.githubusercontent.com/2852703/126125908-e41cfd55-ca26-4197-9b85-688d3a00efe9.png">

Current PR doesn't fix:
- The "reason" column uses up the remaining width

This is because current T-grid implementation isn't responsive and the `reason column` gets cut off in small screen resolutions. If we make reason column use the remaining width, the column is cut off even in bigger screen resolutions. There is a temporary fix for this, but we will hold off for now.

> I'm not sure how this change might impact the current implementation without some thorough desk testing, so my inclination is to hold off on this change for now, and only revisit it if the transition to EuiDataGrid is temporarily delayed.

**How to test**
In order to get alerts in the `Observability > Alerts` page, you should add following configuration:

```
xpack.observability.unsafe.cases.enabled: true
xpack.observability.unsafe.alertingExperience.enabled: true
xpack.ruleRegistry.write.enabled: true
```

Then go into APM, set up some rules with sensitive thresholds, and you should be able to get some alerts. Setting up a log threshold rule will also produce some alerts.
